editor/editor_node.cpp

```diff
@@ -5756,6 +5756,10 @@ EditorNode::EditorNode() {
 	// Exporters might need the theme
 	theme = create_custom_theme();
 
+#ifdef WINDOWS_ENABLED
+	DisplayServer::get_singleton()->cursor_set_custom_image(theme->get_icon("ToolPan", "EditorIcons"), DisplayServer::CURSOR_DRAG);
```

Comment:
I would add a comment to explain why this is done (Windows has no builtin DRAG cursor).

Reply:
Thanks, fixed.